spec/reek/smells/nested_iterators_spec.rb

```diff
@@ -271,6 +271,15 @@ def bad(fred)
       expect(src).to reek_of(:NestedIterators, count: 2).with_config(config)
     end
   end
+
+  it 'should analyze safe-navigation-operator without error' do
+    src = '
+      def f
+        x&.tap {|_| }
+      end
+    '
+    expect(src).not_to reek_of(:NestedIterators, count: 2)
```

Comment:
That said, it may be useful to to have some newer code in our smelly samples feature.